src/Windowing/Silk.NET.SDL/SdlContext.cs

```diff
@@ -102,7 +103,34 @@ public void Dispose()
         public nint GetProcAddress(string proc, int? slot = default)
         {
             AssertCreated();
-            return (nint) _sdl.GLGetProcAddress(proc);
+            _sdl.ClearError();
+            var ret = (nint) _sdl.GLGetProcAddress(proc);
+            _sdl.ThrowError();
+            if (ret == 0)
+            {
+                throw new SymbolLoadingException(proc);
+            }
```

Comment:
not sure, but inlining?

Reply:
Doubt this would be inlined, but wouldn't hurt to fix.